File: edukate-backend/src/main/java/io/github/sanyavertolet/edukate/backend/dtos/BundleMetadata.java
package io.github.sanyavertolet.edukate.backend.dtos;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

@Data
@AllArgsConstructor
public class BundleMetadata {
    private String name;
    private String description;
    private List<String> admins;
    private String shareCode;
    private Boolean isPublic;
    private Long size;
}

File: edukate-backend/src/main/kotlin/io/github/sanyavertolet/edukate/backend/dtos/BundleMetadata.kt
package io.github.sanyavertolet.edukate.backend.dtos

data class BundleMetadata(
    val name: String,
    val description: String,
    val admins: List<String>,
    val shareCode: String,
    val isPublic: Boolean,
    val size: Long,
)
